[0.18.0] Music playback fails and causes the app to crash

### Describe the bug

In version 0.18.0, attempting to play any music files within the app triggers an error, causing the app to crash. This issue affects all types of music files and appears to be specific to the current version.

**Note:** Music playback functions correctly in version 0.17.9, with no issues or crashes observed.

### Logs

### Logs

***Stack Trace***: 
```log
android.view.InflateException: Binary XML file line #214 in org.jellyfin.androidtv:layout/fragment_audio_now_playing: Binary XML file line #214 in org.jellyfin.androidtv:layout/fragment_audio_now_playing: Error inflating class <unknown>
	Suppressed: kotlinx.coroutines.internal.DiagnosticCoroutineContextException: [StandaloneCoroutine{Cancelling}@b6ccc9a, Dispatchers.Main.immediate]
Caused by: android.view.InflateException: Binary XML file line #214 in org.jellyfin.androidtv:layout/fragment_audio_now_playing: Error inflating class <unknown>
Caused by: java.lang.reflect.InvocationTargetException
	at java.lang.reflect.Constructor.newInstance0(Native Method)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:343)
	at android.view.LayoutInflater.createView(LayoutInflater.java:852)
	at android.view.LayoutInflater.createView(LayoutInflater.java:774)
	at com.android.internal.policy.PhoneLayoutInflater.onCreateView(PhoneLayoutInflater.java:58)
	at android.view.LayoutInflater.onCreateView(LayoutInflater.java:928)
	at android.view.LayoutInflater.onCreateView(LayoutInflater.java:948)
	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:1002)
	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:959)
	at android.view.LayoutInflater.rInflate(LayoutInflater.java:1121)
	at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:1082)
	at android.view.LayoutInflater.rInflate(LayoutInflater.java:1124)
	at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:1082)
	at android.view.LayoutInflater.rInflate(LayoutInflater.java:1124)
	at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:1082)
	at android.view.LayoutInflater.inflate(LayoutInflater.java:680)
	at android.view.LayoutInflater.inflate(LayoutInflater.java:532)
	at org.jellyfin.androidtv.databinding.FragmentAudioNowPlayingBinding.inflate(FragmentAudioNowPlayingBinding.java:158)
	at org.jellyfin.androidtv.ui.playback.AudioNowPlayingFragment.onCreateView(AudioNowPlayingFragment.java:98)
	at androidx.fragment.app.Fragment.performCreateView(Fragment.java:3119)
	at androidx.fragment.app.FragmentStateManager.createView(FragmentStateManager.java:577)
	at androidx.fragment.app.FragmentStateManager.moveToExpectedState(FragmentStateManager.java:286)
	at androidx.fragment.app.FragmentManager.executeOpsTogether(FragmentManager.java:2214)
	at androidx.fragment.app.FragmentManager.removeRedundantOperationsAndExecute(FragmentManager.java:2115)
	at androidx.fragment.app.FragmentManager.execSingleAction(FragmentManager.java:2002)
	at androidx.fragment.app.BackStackRecord.commitNow(BackStackRecord.java:317)
	at org.jellyfin.androidtv.ui.browsing.DestinationFragmentView.activateHistoryEntry(DestinationFragmentView.kt:159)
	at org.jellyfin.androidtv.ui.browsing.DestinationFragmentView.navigate(DestinationFragmentView.kt:94)
	at org.jellyfin.androidtv.ui.browsing.MainActivity.handleNavigationAction(MainActivity.kt:116)
	at org.jellyfin.androidtv.ui.browsing.MainActivity.access$handleNavigationAction(MainActivity.kt:33)
	at org.jellyfin.androidtv.ui.browsing.MainActivity$onCreate$2.invokeSuspend(MainActivity.kt:66)
	at org.jellyfin.androidtv.ui.browsing.MainActivity$onCreate$2.invoke(Unknown Source:8)
	at org.jellyfin.androidtv.ui.browsing.MainActivity$onCreate$2.invoke(Unknown Source:4)
	at kotlinx.coroutines.flow.FlowKt__TransformKt$onEach$$inlined$unsafeTransform$1$2.emit(Emitters.kt:50)
	at kotlinx.coroutines.flow.FlowKt__ChannelsKt.emitAllImpl$FlowKt__ChannelsKt(Channels.kt:33)
	at kotlinx.coroutines.flow.FlowKt__ChannelsKt.access$emitAllImpl$FlowKt__ChannelsKt(Channels.kt:1)
	at kotlinx.coroutines.flow.FlowKt__ChannelsKt$emitAllImpl$1.invokeSuspend(Unknown Source:14)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:101)
	at kotlinx.coroutines.EventLoop.processUnconfinedEvent(EventLoop.common.kt:65)
	at kotlinx.coroutines.DispatchedTaskKt.resumeUnconfined(DispatchedTask.kt:231)
	at kotlinx.coroutines.DispatchedTaskKt.dispatch(DispatchedTask.kt:149)
	at kotlinx.coroutines.CancellableContinuationImpl.dispatchResume(CancellableContinuationImpl.kt:470)
	at kotlinx.coroutines.CancellableContinuationImpl.resumeImpl$kotlinx_coroutines_core(CancellableContinuationImpl.kt:504)
	at kotlinx.coroutines.CancellableContinuationImpl.resumeImpl$kotlinx_coroutines_core$default(CancellableContinuationImpl.kt:493)
	at kotlinx.coroutines.CancellableContinuationImpl.resumeWith(CancellableContinuationImpl.kt:359)
	at kotlinx.coroutines.flow.SharedFlowImpl.tryEmit(SharedFlow.kt:414)
	at org.jellyfin.androidtv.ui.navigation.NavigationRepositoryImpl.navigate(NavigationRepository.kt:83)
	at org.jellyfin.androidtv.ui.navigation.NavigationRepository$DefaultImpls.navigate(NavigationRepository.kt:26)
	at org.jellyfin.androidtv.ui.navigation.NavigationRepositoryImpl.navigate(NavigationRepository.kt:62)
	at org.jellyfin.androidtv.ui.playback.rewrite.RewriteMediaManager.playNow(RewriteMediaManager.kt:226)
	at org.jellyfin.androidtv.ui.itemdetail.ItemListFragment.play(ItemListFragment.java:375)
	at org.jellyfin.androidtv.ui.itemdetail.ItemListFragment.play(ItemListFragment.java:354)
	at org.jellyfin.androidtv.ui.itemdetail.ItemListFragment.access$1000(ItemListFragment.java:68)
	at org.jellyfin.androidtv.ui.itemdetail.ItemListFragment$10.onClick(ItemListFragment.java:386)
	at android.view.View.performClick(View.java:7448)
	at android.view.View.performClickInternal(View.java:7425)
	at android.view.View.onKeyUp(View.java:15060)
	at android.view.KeyEvent.dispatch(KeyEvent.java:2867)
	at android.view.View.dispatchKeyEvent(View.java:14247)
	at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1954)
	at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1959)
	at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1959)
	at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1959)
	at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1959)
	at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1959)
	at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1959)
	at android.widget.ScrollView.dispatchKeyEvent(ScrollView.java:491)
	at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1959)
	at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1959)
	at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1959)
	at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1959)
	at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1959)
	at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1959)
	at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1959)
	at com.android.internal.policy.DecorView.superDispatchKeyEvent(DecorView.java:476)
	at com.android.internal.policy.PhoneWindow.superDispatchKeyEvent(PhoneWindow.java:1861)
	at android.app.Activity.dispatchKeyEvent(Activity.java:4085)
	at androidx.core.app.ComponentActivity.superDispatchKeyEvent(ComponentActivity.kt:103)
	at androidx.core.view.KeyEventDispatcher.dispatchKeyEvent(KeyEventDispatcher.java:85)
	at androidx.core.app.ComponentActivity.dispatchKeyEvent(ComponentActivity.kt:117)
	at org.jellyfin.androidtv.ui.browsing.MainActivity.dispatchKeyEvent(MainActivity.kt:174)
	at com.android.internal.policy.DecorView.dispatchKeyEvent(DecorView.java:390)
	at android.view.ViewRootImpl$ViewPostImeInputStage.processKeyEvent(ViewRootImpl.java:5960)
	at android.view.ViewRootImpl$ViewPostImeInputStage.onProcess(ViewRootImpl.java:5828)
	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:5310)
	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:5367)
	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:5333)
	at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:5498)
	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:5341)
	at android.view.ViewRootImpl$AsyncInputStage.apply(ViewRootImpl.java:5555)
	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:5314)
	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:5367)
	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:5333)
	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:5341)
	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:5314)
	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:5367)
	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:5333)
	at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:5531)
	at android.view.ViewRootImpl$ImeInputStage.onFinishedInputEvent(ViewRootImpl.java:5689)
	at android.view.inputmethod.InputMethodManager$PendingEvent.run(InputMethodManager.java:3287)
	at android.view.inputmethod.InputMethodManager.invokeFinishedInputEventCallback(InputMethodManager.java:2829)
	at android.view.inputmethod.InputMethodManager.finishedInputEvent(InputMethodManager.java:2820)
	at android.view.inputmethod.InputMethodManager$ImeInputEventSender.onInputEventFinished(InputMethodManager.java:3264)
	at android.view.InputEventSender.dispatchInputEventFinished(InputEventSender.java:143)
	at android.os.MessageQueue.nativePollOnce(Native Method)
	at android.os.MessageQueue.next(MessageQueue.java:335)
	at android.os.Looper.loop(Looper.java:183)
	at android.app.ActivityThread.main(ActivityThread.java:7676)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
Caused by: android.content.res.Resources$NotFoundException: Drawable org.jellyfin.androidtv:drawable/audio_now_playing_album_background with resource ID #0x7f080064
Caused by: android.content.res.Resources$NotFoundException: File res/kh.jpg from drawable resource ID #0x7f0801a8: .xml extension required
	at android.content.res.ResourcesImpl.loadComplexColorForCookie(ResourcesImpl.java:1210)
	at android.content.res.ResourcesImpl.loadComplexColorFromName(ResourcesImpl.java:1028)
	at android.content.res.ResourcesImpl.loadColorStateList(ResourcesImpl.java:1107)
	at android.content.res.Resources.loadColorStateList(Resources.java:1139)
	at android.content.res.TypedArray.getColorStateList(TypedArray.java:600)
	at android.graphics.drawable.GradientDrawable.updateGradientDrawableSolid(GradientDrawable.java:1701)
	at android.graphics.drawable.GradientDrawable.applyThemeChildElements(GradientDrawable.java:1529)
	at android.graphics.drawable.GradientDrawable.applyTheme(GradientDrawable.java:1439)
	at android.content.res.ResourcesImpl.loadDrawable(ResourcesImpl.java:688)
	at android.content.res.Resources.loadDrawable(Resources.java:993)
	at android.content.res.TypedArray.getDrawableForDensity(TypedArray.java:1006)
	at android.content.res.TypedArray.getDrawable(TypedArray.java:981)
	at android.view.View.<init>(View.java:5535)
	at android.view.ViewGroup.<init>(ViewGroup.java:697)
	at android.widget.FrameLayout.<init>(FrameLayout.java:99)
	at android.widget.FrameLayout.<init>(FrameLayout.java:94)
	at android.widget.FrameLayout.<init>(FrameLayout.java:89)
	at java.lang.reflect.Constructor.newInstance0(Native Method)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:343)
	at android.view.LayoutInflater.createView(LayoutInflater.java:852)
	at android.view.LayoutInflater.createView(LayoutInflater.java:774)
	at com.android.internal.policy.PhoneLayoutInflater.onCreateView(PhoneLayoutInflater.java:58)
	at android.view.LayoutInflater.onCreateView(LayoutInflater.java:928)
	at android.view.LayoutInflater.onCreateView(LayoutInflater.java:948)
	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:1002)
	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:959)
	at android.view.LayoutInflater.rInflate(LayoutInflater.java:1121)
	at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:1082)
	at android.view.LayoutInflater.rInflate(LayoutInflater.java:1124)
	at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:1082)
	at android.view.LayoutInflater.rInflate(LayoutInflater.java:1124)
	at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:1082)
	at android.view.LayoutInflater.inflate(LayoutInflater.java:680)
	at android.view.LayoutInflater.inflate(LayoutInflater.java:532)
	at org.jellyfin.androidtv.databinding.FragmentAudioNowPlayingBinding.inflate(FragmentAudioNowPlayingBinding.java:158)
	at org.jellyfin.androidtv.ui.playback.AudioNowPlayingFragment.onCreateView(AudioNowPlayingFragment.java:98)
	at androidx.fragment.app.Fragment.performCreateView(Fragment.java:3119)
	at androidx.fragment.app.FragmentStateManager.createView(FragmentStateManager.java:577)
	at androidx.fragment.app.FragmentStateManager.moveToExpectedState(FragmentStateManager.java:286)
	at androidx.fragment.app.FragmentManager.executeOpsTogether(FragmentManager.java:2214)
	at androidx.fragment.app.FragmentManager.removeRedundantOperationsAndExecute(FragmentManager.java:2115)
	at androidx.fragment.app.FragmentManager.execSingleAction(FragmentManager.java:2002)
	at androidx.fragment.app.BackStackRecord.commitNow(BackStackRecord.java:317)
	at org.jellyfin.androidtv.ui.browsing.DestinationFragmentView.activateHistoryEntry(DestinationFragmentView.kt:159)
	at org.jellyfin.androidtv.ui.browsing.DestinationFragmentView.navigate(DestinationFragmentView.kt:94)
	at org.jellyfin.androidtv.ui.browsing.MainActivity.handleNavigationAction(MainActivity.kt:116)
	at org.jellyfin.androidtv.ui.browsing.MainActivity.access$handleNavigationAction(MainActivity.kt:33)
	at org.jellyfin.androidtv.ui.browsing.MainActivity$onCreate$2.invokeSuspend(MainActivity.kt:66)
	at org.jellyfin.androidtv.ui.browsing.MainActivity$onCreate$2.invoke(Unknown Source:8)
	at org.jellyfin.androidtv.ui.browsing.MainActivity$onCreate$2.invoke(Unknown Source:4)
	at kotlinx.coroutines.flow.FlowKt__TransformKt$onEach$$inlined$unsafeTransform$1$2.emit(Emitters.kt:50)
	at kotlinx.coroutines.flow.FlowKt__ChannelsKt.emitAllImpl$FlowKt__ChannelsKt(Channels.kt:33)
	at kotlinx.coroutines.flow.FlowKt__ChannelsKt.access$emitAllImpl$FlowKt__ChannelsKt(Channels.kt:1)
	at kotlinx.coroutines.flow.FlowKt__ChannelsKt$emitAllImpl$1.invokeSuspend(Unknown Source:14)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:101)
	at kotlinx.coroutines.EventLoop.processUnconfinedEvent(EventLoop.common.kt:65)
	at kotlinx.coroutines.DispatchedTaskKt.resumeUnconfined(DispatchedTask.kt:231)
	at kotlinx.coroutines.DispatchedTaskKt.dispatch(DispatchedTask.kt:149)
	at kotlinx.coroutines.CancellableContinuationImpl.dispatchResume(CancellableContinuationImpl.kt:470)
	at kotlinx.coroutines.CancellableContinuationImpl.resumeImpl$kotlinx_coroutines_core(CancellableContinuationImpl.kt:504)
	at kotlinx.coroutines.CancellableContinuationImpl.resumeImpl$kotlinx_coroutines_core$default(CancellableContinuationImpl.kt:493)
	at kotlinx.coroutines.CancellableContinuationImpl.resumeWith(CancellableContinuationImpl.kt:359)
	at kotlinx.coroutines.flow.SharedFlowImpl.tryEmit(SharedFlow.kt:414)
	at org.jellyfin.androidtv.ui.navigation.NavigationRepositoryImpl.navigate(NavigationRepository.kt:83)
	at org.jellyfin.androidtv.ui.navigation.NavigationRepository$DefaultImpls.navigate(NavigationRepository.kt:26)
	at org.jellyfin.androidtv.ui.navigation.NavigationRepositoryImpl.navigate(NavigationRepository.kt:62)
	at org.jellyfin.androidtv.ui.playback.rewrite.RewriteMediaManager.playNow(RewriteMediaManager.kt:226)
	at org.jellyfin.androidtv.ui.itemdetail.ItemListFragment.play(ItemListFragment.java:375)
	at org.jellyfin.androidtv.ui.itemdetail.ItemListFragment.play(ItemListFragment.java:354)
	at org.jellyfin.androidtv.ui.itemdetail.ItemListFragment.access$1000(ItemListFragment.java:68)
	at org.jellyfin.androidtv.ui.itemdetail.ItemListFragment$10.onClick(ItemListFragment.java:386)
	at android.view.View.performClick(View.java:7448)
	at android.view.View.performClickInternal(View.java:7425)
	at android.view.View.onKeyUp(View.java:15060)
	at android.view.KeyEvent.dispatch(KeyEvent.java:2867)
	at android.view.View.dispatchKeyEvent(View.java:14247)
	at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1954)
	at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1959)
	at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1959)
	at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1959)
	at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1959)
	at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1959)
	at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1959)
	at android.widget.ScrollView.dispatchKeyEvent(ScrollView.java:491)
	at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1959)
	at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1959)
	at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1959)
	at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1959)
	at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1959)
	at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1959)
	at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1959)
	at com.android.internal.policy.DecorView.superDispatchKeyEvent(DecorView.java:476)
	at com.android.internal.policy.PhoneWindow.superDispatchKeyEvent(PhoneWindow.java:1861)
	at android.app.Activity.dispatchKeyEvent(Activity.java:4085)
	at androidx.core.app.ComponentActivity.superDispatchKeyEvent(ComponentActivity.kt:103)
	at androidx.core.view.KeyEventDispatcher.dispatchKeyEvent(KeyEventDispatcher.java:85)
	at androidx.core.app.ComponentActivity.dispatchKeyEvent(ComponentActivity.kt:117)
	at org.jellyfin.androidtv.ui.browsing.MainActivity.dispatchKeyEvent(MainActivity.kt:174)
	at com.android.internal.policy.DecorView.dispatchKeyEvent(DecorView.java:390)
	at android.view.ViewRootImpl$ViewPostImeInputStage.processKeyEvent(ViewRootImpl.java:5960)
	at android.view.ViewRootImpl$ViewPostImeInputStage.onProcess(ViewRootImpl.java:5828)
	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:5310)
	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:5367)
	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:5333)
	at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:5498)
	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:5341)
	at android.view.ViewRootImpl$AsyncInputStage.apply(ViewRootImpl.java:5555)
	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:5314)
	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:5367)
	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:5333)
	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:5341)
	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:5314)
	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:5367)
	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:5333)
	at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:5531)
	at android.view.ViewRootImpl$ImeInputStage.onFinishedInputEvent(ViewRootImpl.java:5689)
	at android.view.inputmethod.InputMethodManager$PendingEvent.run(InputMethodManager.java:3287)
	at android.view.inputmethod.InputMethodManager.invokeFinishedInputEventCallback(InputMethodManager.java:2829)
	at android.view.inputmethod.InputMethodManager.finishedInputEvent(InputMethodManager.java:2820)
	at android.view.inputmethod.InputMethodManager$ImeInputEventSender.onInputEventFinished(InputMethodManager.java:3264)
	at android.view.InputEventSender.dispatchInputEventFinished(InputEventSender.java:143)
	at android.os.MessageQueue.nativePollOnce(Native Method)
	at android.os.MessageQueue.next(MessageQueue.java:335)
	at android.os.Looper.loop(Looper.java:183)
	at android.app.ActivityThread.main(ActivityThread.java:7676)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)

```  
***Logcat***: 
```log
12-05 11:23:40.212 E/ACRA    (11102): 	at android.view.LayoutInflater.inflate(LayoutInflater.java:532)
12-05 11:23:40.212 E/ACRA    (11102): 	at org.jellyfin.androidtv.databinding.FragmentAudioNowPlayingBinding.inflate(FragmentAudioNowPlayingBinding.java:158)
12-05 11:23:40.212 E/ACRA    (11102): 	at org.jellyfin.androidtv.ui.playback.AudioNowPlayingFragment.onCreateView(AudioNowPlayingFragment.java:98)
12-05 11:23:40.212 E/ACRA    (11102): 	at androidx.fragment.app.Fragment.performCreateView(Fragment.java:3119)
12-05 11:23:40.212 E/ACRA    (11102): 	at androidx.fragment.app.FragmentStateManager.createView(FragmentStateManager.java:577)
12-05 11:23:40.212 E/ACRA    (11102): 	at androidx.fragment.app.FragmentStateManager.moveToExpectedState(FragmentStateManager.java:286)
12-05 11:23:40.212 E/ACRA    (11102): 	at androidx.fragment.app.FragmentManager.executeOpsTogether(FragmentManager.java:2214)
12-05 11:23:40.212 E/ACRA    (11102): 	at androidx.fragment.app.FragmentManager.removeRedundantOperationsAndExecute(FragmentManager.java:2115)
12-05 11:23:40.212 E/ACRA    (11102): 	at androidx.fragment.app.FragmentManager.execSingleAction(FragmentManager.java:2002)
12-05 11:23:40.212 E/ACRA    (11102): 	at androidx.fragment.app.BackStackRecord.commitNow(BackStackRecord.java:317)
12-05 11:23:40.212 E/ACRA    (11102): 	at org.jellyfin.androidtv.ui.browsing.DestinationFragmentView.activateHistoryEntry(DestinationFragmentView.kt:159)
12-05 11:23:40.212 E/ACRA    (11102): 	at org.jellyfin.androidtv.ui.browsing.DestinationFragmentView.navigate(DestinationFragmentView.kt:94)
12-05 11:23:40.212 E/ACRA    (11102): 	at org.jellyfin.androidtv.ui.browsing.MainActivity.handleNavigationAction(MainActivity.kt:116)
12-05 11:23:40.212 E/ACRA    (11102): 	at org.jellyfin.androidtv.ui.browsing.MainActivity.access$handleNavigationAction(MainActivity.kt:33)
12-05 11:23:40.212 E/ACRA    (11102): 	at org.jellyfin.androidtv.ui.browsing.MainActivity$onCreate$2.invokeSuspend(MainActivity.kt:66)
12-05 11:23:40.212 E/ACRA    (11102): 	at org.jellyfin.androidtv.ui.browsing.MainActivity$onCreate$2.invoke(Unknown Source:8)
12-05 11:23:40.212 E/ACRA    (11102): 	at org.jellyfin.androidtv.ui.browsing.MainActivity$onCreate$2.invoke(Unknown Source:4)
12-05 11:23:40.212 E/ACRA    (11102): 	at kotlinx.coroutines.flow.FlowKt__TransformKt$onEach$$inlined$unsafeTransform$1$2.emit(Emitters.kt:50)
12-05 11:23:40.212 E/ACRA    (11102): 	at kotlinx.coroutines.flow.FlowKt__ChannelsKt.emitAllImpl$FlowKt__ChannelsKt(Channels.kt:33)
12-05 11:23:40.212 E/ACRA    (11102): 	at kotlinx.coroutines.flow.FlowKt__ChannelsKt.access$emitAllImpl$FlowKt__ChannelsKt(Channels.kt:1)
12-05 11:23:40.212 E/ACRA    (11102): 	at kotlinx.coroutines.flow.FlowKt__ChannelsKt$emitAllImpl$1.invokeSuspend(Unknown Source:14)
12-05 11:23:40.212 E/ACRA    (11102): 	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
12-05 11:23:40.212 E/ACRA    (11102): 	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:101)
12-05 11:23:40.212 E/ACRA    (11102): 	at kotlinx.coroutines.EventLoop.processUnconfinedEvent(EventLoop.common.kt:65)
12-05 11:23:40.212 E/ACRA    (11102): 	at kotlinx.coroutines.DispatchedTaskKt.resumeUnconfined(DispatchedTask.kt:231)
12-05 11:23:40.212 E/ACRA    (11102): 	at kotlinx.coroutines.DispatchedTaskKt.dispatch(DispatchedTask.kt:149)
12-05 11:23:40.212 E/ACRA    (11102): 	at kotlinx.coroutines.CancellableContinuationImpl.dispatchResume(CancellableContinuationImpl.kt:470)
12-05 11:23:40.212 E/ACRA    (11102): 	at kotlinx.coroutines.CancellableContinuationImpl.resumeImpl$kotlinx_coroutines_core(CancellableContinuationImpl.kt:504)
12-05 11:23:40.212 E/ACRA    (11102): 	at kotlinx.coroutines.CancellableContinuationImpl.resumeImpl$kotlinx_coroutines_core$default(CancellableContinuationImpl.kt:493)
12-05 11:23:40.212 E/ACRA    (11102): 	at kotlinx.coroutines.CancellableContinuationImpl.resumeWith(CancellableContinuationImpl.kt:359)
12-05 11:23:40.212 E/ACRA    (11102): 	at kotlinx.coroutines.flow.SharedFlowImpl.tryEmit(SharedFlow.kt:414)
12-05 11:23:40.212 E/ACRA    (11102): 	at org.jellyfin.androidtv.ui.navigation.NavigationRepositoryImpl.navigate(NavigationRepository.kt:83)
12-05 11:23:40.212 E/ACRA    (11102): 	at org.jellyfin.androidtv.ui.navigation.NavigationRepository$DefaultImpls.navigate(NavigationRepository.kt:26)
12-05 11:23:40.212 E/ACRA    (11102): 	at org.jellyfin.androidtv.ui.navigation.NavigationRepositoryImpl.navigate(NavigationRepository.kt:62)
12-05 11:23:40.212 E/ACRA    (11102): 	at org.jellyfin.androidtv.ui.playback.rewrite.RewriteMediaManager.playNow(RewriteMediaManager.kt:226)
12-05 11:23:40.212 E/ACRA    (11102): 	at org.jellyfin.androidtv.ui.itemdetail.ItemListFragment.play(ItemListFragment.java:375)
12-05 11:23:40.212 E/ACRA    (11102): 	at org.jellyfin.androidtv.ui.itemdetail.ItemListFragment.play(ItemListFragment.java:354)
12-05 11:23:40.212 E/ACRA    (11102): 	at org.jellyfin.androidtv.ui.itemdetail.ItemListFragment.access$1000(ItemListFragment.java:68)
12-05 11:23:40.214 E/ACRA    (11102): 	at org.jellyfin.androidtv.ui.itemdetail.ItemListFragment$10.onClick(ItemListFragment.java:386)
12-05 11:23:40.214 E/ACRA    (11102): 	at android.view.View.performClick(View.java:7448)
12-05 11:23:40.214 E/ACRA    (11102): 	at android.view.View.performClickInternal(View.java:7425)
12-05 11:23:40.214 E/ACRA    (11102): 	at android.view.View.onKeyUp(View.java:15060)
12-05 11:23:40.214 E/ACRA    (11102): 	at android.view.KeyEvent.dispatch(KeyEvent.java:2867)
12-05 11:23:40.214 E/ACRA    (11102): 	at android.view.View.dispatchKeyEvent(View.java:14247)
12-05 11:23:40.214 E/ACRA    (11102): 	at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1954)
12-05 11:23:40.214 E/ACRA    (11102): 	at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1959)
12-05 11:23:40.214 E/ACRA    (11102): 	at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1959)
12-05 11:23:40.214 E/ACRA    (11102): 	at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1959)
12-05 11:23:40.214 E/ACRA    (11102): 	at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1959)
12-05 11:23:40.214 E/ACRA    (11102): 	at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1959)
12-05 11:23:40.214 E/ACRA    (11102): 	at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1959)
12-05 11:23:40.214 E/ACRA    (11102): 	at android.widget.ScrollView.dispatchKeyEvent(ScrollView.java:491)
12-05 11:23:40.214 E/ACRA    (11102): 	at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1959)
12-05 11:23:40.214 E/ACRA    (11102): 	at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1959)
12-05 11:23:40.214 E/ACRA    (11102): 	at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1959)
12-05 11:23:40.214 E/ACRA    (11102): 	at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1959)
12-05 11:23:40.214 E/ACRA    (11102): 	at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1959)
12-05 11:23:40.214 E/ACRA    (11102): 	at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1959)
12-05 11:23:40.214 E/ACRA    (11102): 	at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1959)
12-05 11:23:40.214 E/ACRA    (11102): 	at com.android.internal.policy.DecorView.superDispatchKeyEvent(DecorView.java:476)
12-05 11:23:40.214 E/ACRA    (11102): 	at com.android.internal.policy.PhoneWindow.superDispatchKeyEvent(PhoneWindow.java:1861)
12-05 11:23:40.214 E/ACRA    (11102): 	at android.app.Activity.dispatchKeyEvent(Activity.java:4085)
12-05 11:23:40.214 E/ACRA    (11102): 	at androidx.core.app.ComponentActivity.superDispatchKeyEvent(ComponentActivity.kt:103)
12-05 11:23:40.214 E/ACRA    (11102): 	at androidx.core.view.KeyEventDispatcher.dispatchKeyEvent(KeyEventDispatcher.java:85)
12-05 11:23:40.214 E/ACRA    (11102): 	at androidx.core.app.ComponentActivity.dispatchKeyEvent(ComponentActivity.kt:117)
12-05 11:23:40.214 E/ACRA    (11102): 	at org.jellyfin.androidtv.ui.browsing.MainActivity.dispatchKeyEvent(MainActivity.kt:174)
12-05 11:23:40.214 E/ACRA    (11102): 	at com.android.internal.policy.DecorView.dispatchKeyEvent(DecorView.java:390)
12-05 11:23:40.214 E/ACRA    (11102): 	at android.view.ViewRootImpl$ViewPostImeInputStage.processKeyEvent(ViewRootImpl.java:5960)
12-05 11:23:40.214 E/ACRA    (11102): 	at android.view.ViewRootImpl$ViewPostImeInputStage.onProcess(ViewRootImpl.java:5828)
12-05 11:23:40.214 E/ACRA    (11102): 	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:5310)
12-05 11:23:40.214 E/ACRA    (11102): 	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:5367)
12-05 11:23:40.214 E/ACRA    (11102): 	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:5333)
12-05 11:23:40.214 E/ACRA    (11102): 	at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:5498)
12-05 11:23:40.214 E/ACRA    (11102): 	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:5341)
12-05 11:23:40.214 E/ACRA    (11102): 	at android.view.ViewRootImpl$AsyncInputStage.apply(ViewRootImpl.java:5555)
12-05 11:23:40.214 E/ACRA    (11102): 	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:5314)
12-05 11:23:40.214 E/ACRA    (11102): 	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:5367)
12-05 11:23:40.214 E/ACRA    (11102): 	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:5333)
12-05 11:23:40.214 E/ACRA    (11102): 	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:5341)
12-05 11:23:40.214 E/ACRA    (11102): 	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:5314)
12-05 11:23:40.214 E/ACRA    (11102): 	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:5367)
12-05 11:23:40.214 E/ACRA    (11102): 	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:5333)
12-05 11:23:40.214 E/ACRA    (11102): 	at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:5531)
12-05 11:23:40.214 E/ACRA    (11102): 	at android.view.ViewRootImpl$ImeInputStage.onFinishedInputEvent(ViewRootImpl.java:5689)
12-05 11:23:40.214 E/ACRA    (11102): 	at android.view.inputmethod.InputMethodManager$PendingEvent.run(InputMethodManager.java:3287)
12-05 11:23:40.214 E/ACRA    (11102): 	at android.view.inputmethod.InputMethodManager.invokeFinishedInputEventCallback(InputMethodManager.java:2829)
12-05 11:23:40.214 E/ACRA    (11102): 	at android.view.inputmethod.InputMethodManager.finishedInputEvent(InputMethodManager.java:2820)
12-05 11:23:40.214 E/ACRA    (11102): 	at android.view.inputmethod.InputMethodManager$ImeInputEventSender.onInputEventFinished(InputMethodManager.java:3264)
12-05 11:23:40.214 E/ACRA    (11102): 	at android.view.InputEventSender.dispatchInputEventFinished(InputEventSender.java:143)
12-05 11:23:40.214 E/ACRA    (11102): 	at android.os.MessageQueue.nativePollOnce(Native Method)
12-05 11:23:40.214 E/ACRA    (11102): 	at android.os.MessageQueue.next(MessageQueue.java:335)
12-05 11:23:40.214 E/ACRA    (11102): 	at android.os.Looper.loop(Looper.java:183)
12-05 11:23:40.214 E/ACRA    (11102): 	at android.app.ActivityThread.main(ActivityThread.java:7676)
12-05 11:23:40.214 E/ACRA    (11102): 	at java.lang.reflect.Method.invoke(Native Method)
12-05 11:23:40.214 E/ACRA    (11102): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
12-05 11:23:40.214 E/ACRA    (11102): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
12-05 11:23:40.214 D/ACRA    (11102): Building report
12-05 11:23:40.215 D/ACRA    (11102): Starting collectors with priority FIRST
12-05 11:23:40.219 D/ACRA    (11102): Calling collector org.acra.collector.DropBoxCollector

```  



### FFmpeg logs

```shell

```

### Media info of the file

```shell

```

### Application version

0.18.0

### Device information

Android TV Box

### Android version

Android version 11

### Jellyfin server version

10.10.3

--- a/app/src/main/res/drawable/audio_now_playing_album_background.xml
+++ b/app/src/main/res/drawable/audio_now_playing_album_background.xml
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <shape xmlns:android="http://schemas.android.com/apk/res/android"
     android:shape="rectangle">
-    <solid android:color="?attr/defaultBackground" />
+    <solid android:color="@color/not_quite_black" />
     <corners android:radius="?attr/cardRounding" />
 </shape>

